test/parallel/test-fs-mkdir.js

```diff
@@ -172,6 +173,14 @@ if (common.isMainThread && (common.isLinux || common.isOSX)) {
   });
 }
 
+// mkdirp and mkdirSyncp feature detection
+{
+  assert.strictEqual(fs.mkdir[features].recursive, true);
```

Comment:
let's add a test for the `undefined` case?

Reply:
which undefined case? where `features` is undefined?

Reply:
yeah, I think it would be good to demonstrate that this will work fine for feature detection when `util.features` is `undefined`.

```
!!mkdir[require('util').propretyThatDoesntExist] === false
```